moveit_core/transforms/CMakeLists.txt

```diff
@@ -3,13 +3,16 @@ set(MOVEIT_LIB_NAME moveit_transforms)
 add_library(${MOVEIT_LIB_NAME} src/transforms.cpp)
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
 
-target_link_libraries(${MOVEIT_LIB_NAME} ${catkin_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${Boost_LIBRARIES})
-add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
+target_link_libraries(${MOVEIT_LIB_NAME} ${rclcpp_LIBRARIES} ${rmw_implementation_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${Boost_LIBRARIES})
+# add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
+ament_target_dependencies(${MOVEIT_LIB_NAME}
+            tf2_eigen)
+
 
 install(TARGETS ${MOVEIT_LIB_NAME}
-        LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-        ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
-install(DIRECTORY include/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
+        LIBRARY DESTINATION lib
+      ARCHIVE DESTINATION lib)
+install(DIRECTORY include/ DESTINATION include)
 
 # Unit tests
 if(CATKIN_ENABLE_TESTING)
```

Comment:
Let's port the tests at the same time as the main package.

Reply:
Yeap, this is something I should look at, got a few PRs in ros-planning/ with the same pending issue. I'll book some time this weekend and look at it.

Reply:
@mlautman please see https://github.com/ros-planning/moveit2/pull/12/commits/57814c6c233aa4ee228ca0f4704733b51ff596b5